### DISCUSSION TITLE
Replace Jaeger and Zipkin implementation with Opentelemetry

### DISCUSSION THREAD

**Discussion by secustor** [grafana/grafana#40831] [General]
Currently the Jaeger and Zipkin ( b3 )  are supported.   
I would like to see additionally the OTLP ( with w3c propagation )  format supported too.  

AS OTL supports the other two formats and Jaeger has announced that they will deprecate their API in favor of OTLP, IMO it would make sense to drop Jaeger and replace it with Opentelemetry.

https://github.com/open-telemetry/opentelemetry-go

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
